### ISSUE TITLE
Confirming updates made for release 5 have made it to the browser?

### ISSUE THREAD

**Issue by lauraclarke** [ebi-ait/hca-ebi-wrangler-central#307]
This can't happen before release 5 has been made in the browser but once it has been we need to confirm the updates made as part of the following tickets were propagated to the browser

- [ ]  https://app.zenhub.com/workspaces/dataset-wrangling-status-5f994cb88e0805001759d2e9/issues/ebi-ait/hca-ebi-wrangler-central/240
- [x]  https://app.zenhub.com/workspaces/dataset-wrangling-status-5f994cb88e0805001759d2e9/issues/ebi-ait/hca-ebi-wrangler-central/72
- [x]  https://app.zenhub.com/workspaces/dataset-wrangling-status-5f994cb88e0805001759d2e9/issues/ebi-ait/hca-ebi-wrangler-central/91
- [x]  https://app.zenhub.com/workspaces/dataset-wrangling-status-5f994cb88e0805001759d2e9/issues/ebi-ait/hca-ebi-wrangler-central/74
- [ ]  https://app.zenhub.com/workspaces/dataset-wrangling-status-5f994cb88e0805001759d2e9/issues/ebi-ait/hca-ebi-wrangler-central/245
- [x] https://app.zenhub.com/workspaces/dataset-wrangling-status-5f994cb88e0805001759d2e9/issues/ebi-ait/hca-ebi-wrangler-central/174
- [ ] https://app.zenhub.com/workspaces/dataset-wrangling-status-5f994cb88e0805001759d2e9/issues/ebi-ait/hca-ebi-wrangler-central/114
- [x] https://app.zenhub.com/workspaces/dataset-wrangling-status-5f994cb88e0805001759d2e9/issues/ebi-ait/hca-ebi-wrangler-central/221

# acceptance criteria

For each of these projects, confirm the exported update (data and/or metadata) is visible in the browser

If it is, great tick the box
If it isn't please resurrect the dataset ticket and add it to the needs update column with information about what update was attempted and didn't make it, Please tag @clairerye in any ticket where this is the case so she can coordinate resolution of these bugs. Only tick the box after those steps are done

**Comment by ofanobilbao** [comment]
@ami-day I have moved the ticket to the "To Do" list so that it doesn't get lost. I will try and tag the primary wranglers that might need checking if the updates made it to the browser.

**Comment by ofanobilbao** [comment]
- #240 is stalled pending bulk updates (Ami): If no objections, @ami-day I'll add labels audr,Internally Blocked and move it to "Needs Update". Maybe I'll explore creating BulkUpdates label

- #72 - Two updates were pushed, curator and publication link (Ray): Looks up to date to me in the browser. @rays22 Could you, please, confirm and if so tick the box in the ticket description?

- #91 - Curator was updated (Marion/Wei): Looks up to date to me in the browser. @Wkt8 Could you, please, confirm and if so tick the box in the ticket description?

- #74 - Errors with file names (Ami): To be checked. Also Mallory appears as the curator. It should be Ami right? @ami-day I don't know how to check if this change made it. Could you, please, confirm and check the box in the ticket description? If the curator needs changing maybe you need to re-open the ticket and add it to the Needs Update pipeline on the wrangling board.

- #245 - Ami appearing as contributor rather than curator (Ami): From the ticket is unclear if updates were pushed. It is not updated in the browser. @ami-day could you, please, clarify? It might be that the ticket was closed without actually pushing the update.

- #174 - v3 to v2 change required (Ami): Looks up to date to me in the browser. @ami-day Could you, please, confirm and check the box in the ticket description?

- #114 - It's on the list but I don't see that any updates were done (not obvious) (Ami) Needs clarification or closing @ami-day to confirm

- #221 - Structure changes of the metadata due to errors in pipelines (Ami): I can't check it myself. @ami-day could you, please, confirm if the changes made it through and if so tick the box in the description of the ticket?

Thanks everyone!

**Comment by ami-day** [comment]
Thanks @ofanobilbao , will have a look at these today

**Comment by ami-day** [comment]
@ofanobilbao I have looked into these and selected the ones I checked that are done.
There are a few outstanding issues:

- The update for this project is blocked until we can make bulk updates in ingest prod., as you had written above: https://app.zenhub.com/workspaces/dataset-wrangling-status-5f994cb88e0805001759d2e9/issues/ebi-ait/hca-ebi-wrangler-central/240 update blocked

- I am still showing up as a contributor for the following project in the data portal, though looking at the project in ingest prod., I am not sure what the specific problem is and how to update it, as my name is associated with the "data curator" label, and looks identical to how the curator info. looks in other exported projects in the contributors tab in ingest: https://contribute.data.humancellatlas.org/projects/detail?uuid=0fd8f918-62d6-4b8b-ac35-4c53dd601f71

- There doesn’t seem to be info anywhere about this project needing an update, it was curated and exported a while ago, not sure why it is on the list, I think it may have been incorrectly listed here? https://app.zenhub.com/workspaces/dataset-wrangling-status-5f994cb88e0805001759d2e9/issues/ebi-ait/hca-ebi-wrangler-central/114
